components/candle/challengeService.ts

```diff
@@ -141,6 +130,20 @@ export abstract class ChallengeRegistry {
         return this.challenges.get(challengeId)
     }
 
+    /**
+     * Returns a list of all challenges unlockables
+     */
+    getChallengesUnlockables() {
+        return [...this.challenges.values()].reduce((acc, challenge) => {
+            if (challenge?.Drops?.length) {
+                challenge?.Drops.forEach(
+                    (drop) => (acc[drop.Id] = challenge.Id),
```

Comment:
This applies to everywhere that the `Drop` of a challenge is used.

With #174  merged, now the `Drop`s stored in challenge `json`s is a list of unlockable `Id`s instead. This change will affect the usage of the `Drop` array.

Reply:
Is should now take that into account